input/docs/cs/ClassifyByAlexnet_cs.md

```diff
@@ -0,0 +1,43 @@
+title: Aplikace konvoluční sítě alexnet pro klasifikaci obrázků
+Description: Použití konvoluční sítě alexnet pro klasifikaci obrázku.
+---
+>Tento dokument pracuje s [obrázkem](/zodoc/assets/img/kytka256.jpg) uloženém v proměnné`A`. Obrázek musí být ve složce, ve které současně pracujete s matlabem. 
+
+![](../media/kytka256.jpg)
+
+# Obecná definice konvoluční neuronové sítě Alexnet
```

Comment:
Respektuj konvence týkající se struktury, na začátek dej  trochu poupravený tento popis (kategorií je přesně 100). Výsledek by byl nejlepší jako subplot: vlevo klasifikovaná květina, vpravo tabulka/graf s 5 nejpravděpodobnějšími klasifikacemi (na prvním místě bude daisy) a jejich %.

Reply:
Snažil jsem se přijít na způsob, jak jednoduše vyplotovat další záznamy z kategorie podle scorovací tabulky, ale na žádný snadný jsem nepřišel. V manuálu jsem to také nenašel - resp samotné vyplotované zobrazení by mnou vymyšleným způsobem bylo cca na 2x tolik řádků než současný kód. Stručně to nevymyslím.
Nebo se počítá s tím, že to bude delší?
Děkuji za odpověď

Reply:
Funkce `classify` ti může vracet 2 věci, label je prostě co si o tom myslí, že je na obrázku.. V tomto případě tam je jenom ta kopretina. Avšak druhá návratová hodnota je vektor o délce 1000 - to je skóre pro každou třídu, do které alexnet může klasifikovat... Takže postup by měl být takový, že vytáhneš 5 nejvyšších skóre (jedna část tabulky)  a jejich indexy v tom score vektoru. Indexy využiješ k vytažení labels z alexnet:

    net.Layers(end).ClassNames([1 10 100]);
    % poslední vrstva je vrstva klasifikační, která má vlastnost ClassNames -> takto vytáhneme názvy tříd na 1. 10. a stém umístění...

Taky ti přijde vhod funkce `maxk`...
Jsou to 3 řádky a potom nějaká konstrukce a zobrazení tabulky.

Reply:
Děkuji za odpověď, moc mi to pomohlo! :)